pkg/kubernetes/persistentvolume/v1alpha1/build.go

```diff
@@ -107,13 +107,21 @@ func (b *Builder) WithCapacityQty(resCapacity resource.Quantity) *Builder {
 
 // WithLocalHostDirectory sets the LocalVolumeSource field of PV with provided hostpath
 func (b *Builder) WithLocalHostDirectory(path string) *Builder {
+	return b.WithFormatLocalHostPath(path, "")
```

Comment:
Do have a look at this. Do we really need a separate build method? This is related to my earlier comment on separating two properties.

I feel, we can have functions that do the combined stuff as well as set each property separately.

Final, Build function(s) will decide on default if any.